build.gradle

```diff
@@ -251,9 +251,9 @@ dependencies {
     testCompile "com.google.jimfs:jimfs:1.1"
 }
 
-//add gatk-launch to the jar as a resource
+//add gatk to the jar as a resource
```

Comment:
"add gatk launcher script to the jar as a resource" (less confusing in this particular context)

Reply:
good catch